### PR TITLE
5849: Fix for Unable to create table language_text and themes due to SQL version

### PR DESCRIPTION
**Pull request by pranjaljn97**
Issue No: 5849
Plese refer to bug report, http://www.atutor.ca/atutor/mantis/view.php?id=5849

### CODE REVIEW

File: install/db/achecker_schema.sql [hunk 1]
```diff
@@ -308,7 +308,7 @@ CREATE TABLE `themes` (
   `title` varchar(80) NOT NULL DEFAULT '',
   `version` varchar(10) NOT NULL DEFAULT '',
   `dir_name` varchar(20) NOT NULL DEFAULT '',
-  `last_updated` date NOT NULL DEFAULT '0000-00-00',
+  `last_updated` date  NULL DEFAULT NULL,
```
Comment (cindyli) [inline install/db/achecker_schema.sql]: It doesn't make sense to have the first "NULL".

**Comment by cindyli** [comment]
@pranjaljn97, this pull request is far from completion. When addressing a problem in the system, we cannot address a small portion of it but leave the rest behind, which introduces the confusion and traps as well as affecting the code quality.

**Comment by pranjaljn97** [comment]
I got your point. I was just trying to address the problem that I faced in the installation on the Local machine.
But there are a lot of things in DB section that needed to be done.
Can you please guide me through the tasks to be done or I should close this PR for now? 
Thanks. @cindyli

**Comment by cindyli** [comment]
If you don't have a clear picture of what needs to be done, please close this PR. Thanks.

**Pull request closed without merging**